### PR TITLE
Fix a few issues in contributing documentation.

### PR DESCRIPTION
**Pull request by cwest**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,16 +50,17 @@ system already, but many of them are preinstalled on a Pivotal dev box.
 
 (Run all of these commands from the project root)
 
-ruby 2.1.2 
+ruby 2.1.2
 
-bundle
+bundler
```
Comment (gpleiss) [inline CONTRIBUTING.md]: This one should be `bundle`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -50,16 +50,17 @@ system already, but many of them are preinstalled on a Pivotal dev box.
...
 
-  $ gem install bundle
-    
-node and npm 
+  $ gem install bundler
+
+node and npm (at this time `nodejs` version `0.10` is required. Install
+[nvm](https://github.com/creationix/nvm) then use it to install node and npm.)
+
+  $ nvm install 0.10
```
Comment (gpleiss) [inline CONTRIBUTING.md]: What issues were you having with other versions of node? If you were having problems with brew's default version, I'm all for locking down our node dependency. However, I'd prefer using a more recent version than 0.10. We're currently using v0.12.2 and haven't run into any problems.

File: CONTRIBUTING.md (hunk 1 above)
Comment (cwest) [inline CONTRIBUTING.md]: Is this meant to be the name of the software (`bundler`) or the name of the CLI command (`bundle`)?

I took it to be the name of the software.

Comment (gpleiss) [inline CONTRIBUTING.md]: Ah you're totally right! My bad :grin:

File: CONTRIBUTING.md (hunk 2 above)
Comment (cwest) [inline CONTRIBUTING.md]: Indeed this was an issue with the brew default nodejs. I inspected the [travis.yml](https://github.com/pivotal-cf/pivotal-ui/blob/master/.travis.yml) file to understand which version(s) of nodejs we're targeting. The project, via its CI configuration, is presently locked to 0.10. This documentation change makes that explicit.

Comment (gpleiss) [inline CONTRIBUTING.md]: Gotcha. That makes sense. We should push contributors to use a more recent version of node, but we can do that down the line after we've vetted a newer version. I'll accept these doc changes as is.

**Comment by gpleiss** [comment]
Thanks @cwest!

**Pull request merged**